### DISCUSSION TITLE
No Bootscreen on MacPro5,1 with RX580

### DISCUSSION THREAD

**Discussion by mark-heiple** [dakanji/RefindPlus#109] [General]
I used MyBootMgr to install RefindPlus, for the purpose of getting a boot screen so that I can dual boot between macOS High Sierra and Linux Mint 20. I'm not interested in installing newer unsupported versions of macOS.

My mac has both a an RX580 and a HD 5770. When the monitor is connected to the HD5770, everything is fine. Plugged into the RX580 - nothing. The numeric keys don't start any OS either.

I had used refind before using MyBootMgr to install refindPlus. I thought I had cleared out all the previous refind files from the EFI volume, buy maybe not.

What specifically needs to be on the EFI partition to get a boot screen with a RX580? I'm wondering if I have the wrong bootx64.efi. Is this what I need to get a boot screen? Does it need to had any specific filename? This is what my /EFI/BOOT looks like:

![efi](https://user-images.githubusercontent.com/27691930/164944633-f08346b4-6f7d-4651-abb0-d5a5b9251b92.jpg)

**Comment by mark-heiple** [comment]
One complication is my system has multiple drives in it, all of which have an EFI partition. Which gets used?

**Comment by dakanji** [comment]
> What specifically needs to be on the EFI partition to get a boot screen with a RX580? I'm wondering if I have the wrong bootx64.efi. Is this what I need to get a boot screen? Does it need to had any specific filename? 

Hi,

These are the questions asked but the answers to them will not resolve your problem. You will be better off making the effort to raise an actual issue and follow the issue template which requests some information, logs specifically, which can be looked at to provide a solution, which might be a fix to RefindPlus itself and a workaround etc.

For the record, the answers to the queries are:

> What specifically needs to be on the EFI partition to get a boot screen with a RX580? 

Nothing specifically as long as the GPU does not have a problem.

> I'm wondering if I have the wrong bootx64.efi

Unless you changed the file placed by MyBootMgr, you have the right file

> Is this what I need to get a boot screen?

Yes it is

> Does it need to had any specific filename?

It does, BOOTx64.efi, as named by MyBootMgr

***

As stated, these will not resolve the problem so please raise an actual issue.
Since you used MyBootMgr to set things up, please also attach the ConfigFactory debug log.

**Comment by startergo** [comment]
If you connect both original Apple card and non-apple cards the boot screen will always be on the Apple card even if it is not connected to a screen.

**Comment by dakanji** [comment]
Come across that mentioned before but as the RefindPlus GPU selection process is independent of the firmware, this could init the other GPU. As the poster appears unwilling/unable to  raise the issue and provide the information required to properly consider this, I suppose they could just disconnect the other GPU.
